apps/public/src/app/(map)/locatie/kies/_components/SideBar/SideBar.tsx

```diff
@@ -10,15 +12,67 @@ type Props = {
   coordinates?: Coordinates
 }
 
+type Address = {
+  id: string
+  weergave_naam: string
+}
+
+const pdokQueryParams =
+  'fq=bron:BAG&fq=type:adres&fq=gemeentenaam:(amsterdam "ouder-amstel" weesp)&fl=id,weergavenaam&rows=5'
```

Comment:
@vdegraaf wrote: Why did you choose 5 options instead of the 10 as designed?

10 seemed like a lot, but I should stick to the design 😁 Changed it!